### PR TITLE
Account card addresses adding/updating

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7233/

**Comment by huwshimi** [comment]
QA:
- visit your account page
- remove your payment method
- add a new one with the "address is the same" checkbox ticked
- the payment method should be created with the same address as the user address
- remove your payment method
- add a new one WITHOUT the "address is the same" checkbox ticked
- the payment method should be created with the new address
- click the button to update the address
- change the address and expiry
- click "Update"
- the address and expiry should update

**Comment by fabricematrat** [comment]
If you try to update the CC the current expiry date has not the right format by default. You click on it and exit to see the issue. 4/24 instead of 04/24.

File: jujugui/static/gui/src/app/components/account/payment-methods/_payment-methods.scss [hunk 1]
```diff
@@ -24,5 +24,13 @@
         margin-left: 20px;
       }
     }
+
+    .inset-select {
+      margin: 20px 0;
+    }
+
+    input[type="checkbox"] {
```
Comment (Lukewh) [inline jujugui/static/gui/src/app/components/account/payment-methods/_payment-methods.scss]: should add a class really

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7253/

**Comment by huwshimi** [comment]
Thanks everyone :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**